eras/alonzo/test-suite/test/Test/Cardano/Ledger/Alonzo/Serialisation/Tripping.hs

```diff
@@ -18,9 +18,9 @@ import Cardano.Ledger.Alonzo.Rules (
 import Cardano.Ledger.Alonzo.Scripts (eqAlonzoScriptRaw)
 import Cardano.Ledger.Block (Block)
 import Cardano.Ledger.Core
-import Cardano.Ledger.Crypto (StandardCrypto)
 import Cardano.Ledger.MemoBytes (zipMemoRawType)
 import Cardano.Ledger.Plutus.Data (BinaryData, Data (..))
+import Cardano.Protocol.Crypto (StandardCrypto)
```

Comment:
We should probably bump the version of cardano-protocol-tpraos  in cardano-ledger-alonzo-test.cabal
to 
`cardano-protocol-tpraos >=1.4,`

Reply:
Done!